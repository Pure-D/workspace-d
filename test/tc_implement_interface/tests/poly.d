module poly;

class Foo : IFoo1, IFoo2
{
}

interface IFoo1 : IFoo
{
	void firstFoo();
}

interface IFoo2 : IFoo
{
	void secondFoo();
}

interface IFoo
{
	void baseFoo();
}
