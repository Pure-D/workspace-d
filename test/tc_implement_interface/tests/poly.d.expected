implement 27
#=1 firstFoo
#=1 secondFoo
#=1 baseFoo
